src/main/java/net/neoforged/neoforge/common/NeoForgeMod.java

```diff
@@ -630,6 +639,14 @@ public NeoForgeMod(IEventBus modEventBus, Dist dist, ModContainer container) {
 
         modEventBus.register(NeoForgeDataMaps.class);
 
+        modEventBus.addListener(AddPackFindersEvent.class, event -> event.addPackFinders(
+                ResourceLocation.fromNamespaceAndPath("neoforge", "data/neoforge/datapacks/mod_experimental"),
+                PackType.SERVER_DATA,
+                Component.translatable("pack.neoforge.experimental.name"),
+                PackSource.FEATURE,
+                false,
+                Pack.Position.BOTTOM));
```

Comment:
Erm... this position seems... wrong. Feature-flag-enabled packs should generally be `Pack.Position.TOP` (like the vanilla ones are)

Reply:
tbh I wasnt aware the feature packs are registered on top all others, this has been fixed